doc/architecture/architecture.rst

```diff
@@ -1,16 +1,190 @@
 .. _architecture:
 
 Ostro |trade| OS Architecture
-##################################
+#############################
 
-The Ostro OS is a Linux*-based operating system tailored for IoT devices and built with
-security in mind.  Ostro OS is based on standard libraries and services with 
-frameworks to help with best practices and common IoT use cases.
+Introduction
+============
 
-The following documents start off our descriptions of the Ostro |trade| OS 
-components and their relationships so developers
-can learn more technical details about what we're building.
+The Ostro |trade| OS is a pre-compiled, configured and secured base
+Internet of Things (IoT) Linux\* OS that supports creating custom images
+easily. For more information, see :ref:`about_ostro`.
 
+This document gives an overview to Ostro OS architecture and the basic
+building blocks. Furthermore, the document briefly talks about how
+Ostro OS meets different end-user use-cases ranging from hacking to
+IoT product creation.
+
+.. _`Yocto Project`: http://yoctoproject.org
+.. _`Soletta Project`: http://solettaproject.org
+.. _`Node.JS`: https://nodejs.org
+.. _`IoTivity`: https://www.iotivity.org/
+
+Ostro |trade| OS Characteristics
+=================================
+
+In Ostro |trade| OS, there are four key characteristics that
+drive the architecture design:
+
+1. Ostro OS uses standard libraries and services: The building blocks
+used in Ostro OS are commonly used in open source and any Ostro OS
+custom implementation is avoided. This has a big benefit because it
+means there is no extra porting/platform adaptation needed to get
```

Comment:
Would it be appropriate to change that to 'very little' as there would still be a little porting needed, if nothing else in the form of a app manifest to be created.